doc/reference/schunk_api.rst

```diff
@@ -19,6 +19,8 @@ Methods
     SChunk.insert_data
     SChunk.update_chunk
     SChunk.update_data
+    SChunk.get_slice
+    SChunk.__setitem__
```

Comment:
Looking at that, I'd add a `SChunk.__getitem__` here too.  You can use `get_slice` under the hood and return a bytes object.

Reply:
Fixed in 20b3d298e5aef2b2731aabf104d51dc0a67dcaca.